docs/source/usage/quick_start.rst

```diff
@@ -88,16 +127,35 @@ To load a saved MetaFrame, use the following code:
 Synthesizing the Data
 ---------------------
 
-With the :obj:`MetaFrame <metasyn.metaframe.MetaFrame>` loaded, you can synthesize new data. To do so, we simply call the :meth:`metasyn.metaframe.MetaFrame.synthesize` function on the :obj:`MetaFrame<metasyn.metaframe.MetaFrame>`, and pass in the number of rows we'd like to generate as a parameter. Let's generate five rows of synthetic data.
+With the :obj:`MetaFrame <metasyn.metaframe.MetaFrame>` loaded, you can synthesize new data. To do so, we simply call the :func:`~metasyn.metaframe.MetaFrame.synthesize` function on the MetaFrame, and pass in the number of rows you'd like to generate as a parameter. For example, to generate five rows of synthetic data we can use: 
```

Comment:
This is a method not a function though?

Reply:
Whoops, sometimes get the two mixed around.